CHANGELOG.md

```diff
@@ -48,6 +50,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 - The `examples` folder has been migrated to the [openEO Community Examples](https://github.com/Open-EO/openeo-community-examples/tree/main/processes) repository.
+- `between`: Support for temporal comparison.
```

Comment:
Did you think of adding a new process for this? I don't know if it is used/needed by anyone but currently our implementation supports this.

Reply:
Not planned right now, but would be something we can add anytime when needed. You can continue to support it.